tests/cli/test_cli_options.py

```diff
@@ -136,6 +148,8 @@ def check_generated_output(gen_expected_path, gen_result_path, result_manip=None
     True
     """
 
+    no_additional_options = bool(program_args.diff is False and program_args.apply is False)
```

Comment:
you can remove this check and replace `if no_additional_options:` with `if not program_args:`